apps/src/templates/progress/progressTypes.js

```diff
@@ -27,9 +38,9 @@ export const lessonType = PropTypes.shape({
   lockable: PropTypes.bool.isRequired,
   stageNumber: PropTypes.number,
   lesson_plan_html_url: PropTypes.string,
-  isFocusArea: PropTypes.bool.isRequired,
```

Comment:
Where did isFocusArea go? Thats needed for PLC

Reply:
progressRedux.js line 441. i changed that and level.isCurrentLevel from shape properties to lookups since neither are widely used.